ci-operator/config/openshift/special-resource-operator/openshift-special-resource-operator-master.yaml

```diff
@@ -0,0 +1,40 @@
+base_images:
+  base:
+    name: "4.7"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.15
+promotion:
+  name: "4.7"
+  namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 4Gi
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tag_specification:
+  name: "4.7"
+  namespace: ocp
+tests:
+- as: e2e-aws
+  commands: TEST_SUITE=openshift/conformance/parallel run-tests
+  openshift_installer:
```

Comment:
Is there any reason why using templates instead of a multi-stage job? Please check the docs first.  [example](https://docs.ci.openshift.org/docs/getting-started/examples/#how-do-i-add-a-job-that-runs-the-openshift-end-to-end-conformance-suite-on-aws)

Reply:
Thanks for the quick response! As far as I know this is just what was automatically created when using `make new-repo`. I will change it to a multi-stage job.

Reply:
Thanks @dagrayvid !